SPDX-FileCopyrightText: 2023 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

usr/<target> is an old, obselete location. Patch it out - we don't
want things there.

diff --color -ru binutils/Makefile.am binutils/Makefile.am
--- binutils/Makefile.am	2023-02-01 16:50:53.438231898 +1100
+++ binutils/Makefile.am	2023-02-01 16:51:30.269712861 +1100
@@ -7,7 +7,7 @@
 
 SUBDIRS = doc po
 
-tooldir = $(exec_prefix)/$(target_alias)
+tooldir = $(exec_prefix)
 
 ## These aren't set by automake, because they appear in
 ## bfd/acinclude.m4, which is included by binutils/acinclude.m4, and
diff --color -ru gas/Makefile.am gas/Makefile.am
--- gas/Makefile.am	2023-02-01 16:50:53.494232626 +1100
+++ gas/Makefile.am	2023-02-01 16:51:33.857759722 +1100
@@ -7,7 +7,7 @@
 
 SUBDIRS = doc po
 
-tooldir = $(exec_prefix)/$(target_alias)
+tooldir = $(exec_prefix)
 
 YACC = `if [ -f ../bison/bison ] ; then echo ../bison/bison -y -L../bison/bison ; else echo @YACC@ ; fi`
 LEX = `if [ -f ../flex/flex ] ; then echo ../flex/flex ; else echo @LEX@ ; fi`
diff --color -ru ld/Makefile.am ld/Makefile.am
--- ld/Makefile.am	2023-02-01 16:50:53.555233423 +1100
+++ ld/Makefile.am	2023-02-01 16:51:38.506820435 +1100
@@ -7,7 +7,7 @@
 
 SUBDIRS = po
 
-tooldir = $(exec_prefix)/$(target_alias)
+tooldir = $(exec_prefix)
 
 YACC = `if [ -f ../bison/bison ]; then echo ../bison/bison -y -L$(srcdir)/../bison/; else echo @YACC@; fi`
 YFLAGS = -d
